packages/spear-cli/README_ja.md

```diff
@@ -95,13 +95,148 @@ spear watch -s <project directory>
 ### サイトマップ生成
 
 `spear-cli` の設定ファイルでサイトマップ生成設定がされている場合、自動でサイトマップを生成します。  
-サイトマップ生成設定には、サイトマップ生成の有無と、ホスト URL の情報が必要です。次の2つの設定を `spear.config.js` に入れることで動作します。
+サイトマップ生成設定には、サイトマップ生成の有無と、ホスト URL の情報が必要です。次の2つの設定を `spear.config.mjs` に入れることで動作します。
 
 ```javascript
   "generateSitemap": boolean,
   "siteURL": string,
 ```
 
+### SEO タグ
+
+SEO タグ設定をすることで、生成した HTM  ファイルに SEO タグを注入することが出来ます。  
+設定は `spear.config.mjs` のプラグイン機能を利用します。
+
+```javascript
+import { spearSEO } from "@spearly/spear-cli/dist/plugins/spear-seo.js"
+export default {
+  ...
+  plugins: [
+    spearSEO(),
+  ]
+}
+```
+
+コンポーネントやページ内で以下のタグを好きな場所に書くことで、SEO タグを自動で注入します。
+
+```html
+<spear-seo
+  title="Page title"
+  meta-description="Page description"
+  meta-og:url="/pics/ogp.png">
+</spear-seo>
+```
+
+指定可能な属性は以下の通りです。
+
+| Attribute | Description | Generated Value|
+|-----------|-------------|----------------|
+| title |	サイトタイトル。 |  <title>value</title> |
+| meta-*** | メタ情報 | <meta name="****" value="value"> |
+| link-*** | リンク情報 | <link rel="***" href="value"> |
+
+また  サイト内で共通のグローバル値  `spearSEO` 関数の引数として指定できます。
+
+```javascript
+import { spearSEO } from "@spearly/spear-cli/dist/plugins/spear-seo.js"
+export default {
+  ...
+  plugins: [
+    spearSEO({
+      "title": "My Blog"
```

Comment:
It would be better to document that what you set here can be called by `#seo_title` 🙏.